### ISSUE TITLE
Chosing a run for the submission based on training or results?

### ISSUE THREAD

**Issue by Newgreen64** [CMM-25/a3#31]
Dear TAs,

we stand before the decision which run we should submit for our walk. We have a nice looking result from before we started with the cartwheel. However, the training for this run was a bit noisy. During our cartwheel we also improved some hyperparameters and train another walk policy with it in the end. This lead to nicer looking training curves, but the videos shows the torso starting to tilt over towards the end of the trajectory. Unfortunately, we do not have the server time remaining for another run,to try to fix this with different hyperparameters. 

The first candidate policy looks like this: 

https://github.com/user-attachments/assets/a4d47d69-c3f7-407b-b542-20582414d01b

The second candidate policy looks like this: 

https://github.com/user-attachments/assets/0a7407c4-bc3a-45eb-b13c-405d0e87b197

In the training data, the noisier training of the first policy (green) is apparent: 

![Image](https://github.com/user-attachments/assets/0a9e09b5-3a0e-46bd-9604-dc8e814ac0cb)

![Image](https://github.com/user-attachments/assets/e3d11cae-3fa0-4c3a-8fb7-457374ea63af)

(The different magnitude of rewards is also caused by changed hyperparameters of the reward functions) 

We are a bit unsure how the runs will be evaluated and accordingly which one we should submit. Any advise would be much appreciated. 

Kind regards,
Simon

**Comment by fzargarbashi** [comment]
Hi,
I'd say having good policy that generates nice motion results is first priority.

Best,
Fatemeh

**Comment by Newgreen64** [comment]
Alright, thank you!
